src/components/application_manager/test/commands/mobile/delete_command_request_test.cc

```diff
@@ -221,7 +223,9 @@ TEST_F(DeleteCommandRequestTest,
       .WillByDefault(Return(am::HmiInterfaces::STATE_AVAILABLE));
   ON_CALL(*app, FindCommand(kCommandId)).WillByDefault(Return(test_msg.get()));
   ON_CALL(*app, get_grammar_id()).WillByDefault(Return(kConnectionKey));
-
+  EXPECT_CALL(app_mngr_, GetRPCService())
```

Comment:
@ValeriiMalkov please replace with ON_CALL.
Try to avoid using EXPECT_CALLS for non critical for use cases getters. 
It is always smells a little bit bad is there is multiple Times expectations.

Reply:
@LuxoftAKutsan @LitvinenkoIra fixed https://github.com/LitvinenkoIra/sdl_core/pull/3/commits/99eb6746366c2ae8bff7be8fdc67dc300f2c5330